monitoring/README.md

```diff
@@ -92,47 +157,113 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
+   - You can also use the following combination of `NS/APIService` values to monitor the APIserver of the hosting cluster or a WEC cluster: `NS: default` and `APIService: apiserver`
+   - Use the following commands to obtain the Ip address of the APIServer in the hosting cluster, WDS (e.g., `wds1`) and ITS (e.g., `its1-system`) spaces:
+     ```bash
+     kubectl --context <hosting-cluster-context> get Endpoints -A | egrep "NAME| wds| vcluster |kubernetes"
+     ```
+     For example: 
+     ```bash
+     $ kubectl --context kind-kubeflex get Endpoints -A | egrep "NAME| wds| vcluster |kubernetes"
+     ```
+     Output:
+     ```bash
+     NAMESPACE         NAME                                                              ENDPOINTS                                                AGE
+     default           kubernetes                                                        172.18.0.2:6443                                          3d20h
+     its1-system       vcluster                                                          10.244.0.12:8443,10.244.0.12:8443                        3d20h
+     wds1-system       wds1                                                              10.244.0.21:9444                                         3d20h
+     ```
+     
+     If you're using kind clusters then use the following command to obtain the IP address of the host container for your cluster: `$ docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' <docker-container-name>`
+
+     For example: 
+     ```bash
+     docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' cluster1-control-plane
+     ```
+
+     Output:
+     ```bash
+       172.18.0.3
+     ```
+
+Here are the default port number for the APIservers: 
+
+| APIServer Pod | Default Port #|
+| ------------- | ------------- |
+| Host-cluster  |     6443      |
+| WDS space     |     6443      |
+| ITS space     |     8443      |
+| WEC-cluster   |     9444      |
 
-Note: you can also monitor the APIserver of the hosting cluster using the following values: `NS: default` and `APIService: apiserver`
 
 b) API Server Priority & Fairness: input into the following text boxes.
 - `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, *its1-system*, etc.
 - `APIService`: name of the service for a KubeStellar space APIserver, e.g., *wds1*, *wds2*, *vcluster*, etc.
+- `Instance`: hostname and port number for the target KubeStellar space APIserver, e.g., *10.244.0.18:9444*
 
     <img src="images/apiserver-apf-monitoring.png" width="60%" height="80%" title="apiserver-apf-monitoring.png"> 
 
-Note: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space or the hosting cluster. 
+**Note**: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space, the hosting or WEC clusters. 
```

Comment:
`instance` too, right?

Reply:
Yes, but the instance value is not constant - it's environment dependent. These instructions just explain the combination of constant values that the user should or must use.

Reply:
But the user needs to know what value to use for the "instance" variable. Something should be said about that. If I understand correctly, the same combination of `NS`, `APIService`, and `Instance` that works for a given target in 5-a also works for the same target in 5-b. Couldn't the instructions say something that simple, without getting tangled up in the fact that the discussion in 5-a is structured as a table plus some other comments?

Reply:
I created an issue to keep track of this here: https://github.com/kubestellar/kubestellar/issues/2470 - I will address it in a new PR. Thanks!